### PR TITLE
(fixes #203) add mozilla wordmark to footer

### LINKED ISSUE #203: place Mozilla logo mark on bottom left side of footer element

### PR DESCRIPTION
**Pull request by spasovski**
image

### CODE REVIEW

File: src/components/icons/MozillaLogo.svelte
Comment (hamilton) [inline src/components/icons/MozillaLogo.svelte]: I'd remove `height` from this and only allow the user to specify width, if possible, to prevent warp

**Review by hamilton — CHANGES_REQUESTED** [review]
lgtm, one minor nit about props, once you take a look feel free to merge at your convenience

**Comment by spasovski** [comment]
ready for re-r when you get a chance

**Review by hamilton — APPROVED** [review]
looks good @spasovski – thanks!

**Pull request merged**
